### DISCUSSION TITLE
Start & End support for multi-directional layouts

### DISCUSSION THREAD

**Discussion by SakarDev** [tailwindlabs/tailwindcss#9187] [Ideas]
If there was an option to use **margin-start** or **margin-end** instead of using margin-left or margin-right it would save us a lot of time.
Trust me in big projects managing RTL support is so much boring.

Instead of writing this:
`<div class="ltr:ml-3  rtl:mr-3">....</div>`

We would have to write only this:
`<div class="ms-2">....</div>`

**Comment by onemoreahmad** [comment]
Agree +1 

same as tainlindcss-rtl package does https://github.com/20lives/tailwindcss-rtl

**Comment by adamwathan** [comment]
Check the existing discussion here: https://github.com/tailwindlabs/tailwindcss/discussions/1483
